src/ansys/dpf/core/examples/examples.py

```diff
@@ -422,3 +422,39 @@ def find_distributed_msup_folder(
 
     """
     return find_files(distributed_msup_folder, should_upload, server, return_local_path)
+
+
+def find_fluent_model(should_upload: bool = True, server=None, return_local_path=False) -> str:
```

Comment:
@rafacanton aren't those methods supposed to use `find_files`?

Reply:
@rafacanton you can also add it to the tests by appending the `list_examples` in `test_examples.py/l.87`